mod_deploy/controllers.py

```diff
@@ -8,6 +8,7 @@
 import hmac
 import json
 import subprocess
+import os
```

Comment:
Is it needed outside the `deploy()` function?

Reply:
Nope.